guides/micronaut-email-amazon-ses/micronaut-email-amazon-ses.adoc

```diff
@@ -0,0 +1,66 @@
+external:micronaut-email/header.adoc[]
+
+external:micronaut-email/controller.adoc[]
+
+external:micronaut-email/configuration.adoc[]
+
+=== AWS SES
+
+____
+Amazon Simple Email Service (Amazon SES) is a cloud-based email sending service designed to help digital marketers and application developers send marketing, notification, and transactional emails. It is a reliable, cost-effective service for businesses of all sizes that use email to keep in contact with their customers.
+____
+
+==== Dependency
+
+Because we added `email-ses` feature, the application contains the following dependency:
+
+dependency:micronaut-email-ses[groupId=com.micronaut.email]
+
+=== Micronaut AWS SDK v2
+
+Micronaut Amazon SES integration uses https://micronaut-projects.github.io/micronaut-aws/latest/guide/#sdkv2[Micronaut AWS SDK v2 integration].
```

Comment:
```suggestion
The Micronaut Amazon SES integration currently uses https://micronaut-projects.github.io/micronaut-aws/latest/guide/#sdkv2[Version 2].
```

Reply:
I don't think this sounds as what I intended.